scripts/e2e/hack/ovf_import_and_deploy.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
+
+"""
+* *******************************************************
+* Copyright VMware, Inc. 2016, 2018. All Rights Reserved.
```

Comment:
Note that we are also close to content library support in govmomi/govc, when that's done we should be able to avoid the python dep here: https://github.com/vmware/govmomi/issues/735#issuecomment-451690562

Reply:
I think we all came to agreement yesterday to remove the python code and wait for the govc support.  I added a hold below (which Hui can unhold).  This should resolve all my comments about the python code.